examples/Server/Startup.cs

```diff
@@ -27,14 +29,28 @@ public class Startup
         // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddGrpc();
+            services
+                .AddGrpc(options =>
+                {
+                    options.AddInterceptor<LoggingInterceptor>();
+                })
+                .AddServiceOptions<MailerService>(options =>
+                {
+                    options.AddInterceptor<DuplexStatisticsInterceptor>();
+                })
+                .AddServiceOptions<GreeterService>(options =>
+                 {
+                     options.AddInterceptor<GreetingCachingInterceptor>();
+                 });
             services.AddSingleton<IncrementingCounter>();
             services.AddSingleton<MailQueueRepository>();
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
```

Comment:
Why did you add this?

Reply:
I was using it to test interceptors at one point